modules/ROOT/partials/plugin-apis/comments-apis.adoc

```diff
@@ -56,13 +56,17 @@ The `getEventLog` returns a log that contains information and timestamps of all
 ----
 {
   "type": "reply",
-  "timestamp": "2024-10-01T03:07:53.771Z",
-  "conversationUid": "annotation-r1nn5xdo5ye",
-  "commentUid": "annotation-uh00rb41kma",
+  "timestamp": "2024-12-06T06:38:42.444Z",
+  "conversationUid": "mce-conversation_16214546531733467122444",
```

Comment:
To keep consistent with the rest of example data, please revert the conversationUid

Reply:
```suggestion
  "conversationUid": "annotation-r1nn5xdo5ye",
```